moj_pokusaj_smart_home/meteo.py

```diff
@@ -0,0 +1,34 @@
+import requests, json
+import xmltodict
+
+url = "https://vrijeme.hr/hrvatska_n.xml"
+
+response = requests.get(url)
+
+data = xmltodict.parse(response.content)
+
+grad = "Zagreb-Maksimir"
+
+def vrijeme():
+    if response.status_code == 200:   
+        data = response.json()   
```

Comment:
ovo nam baš i ne radi jel tako? Podaci nam nisu JSON u ovom slučaju i čini mi se da nekako napamet pišeš bez da zapravo razumiješ o čemu se radi - na liniji 8 imamo `data = xmltodict.parse(response.content)` i tu smo već pretvorili podatke sa API-ja u python objekt (konkretno u dictionary) i ti u ovoj liniji pokušavaš napraviti to isto (samo sa `.json`) zato što smo većinu puta dosad radili sa API-jima koji vraćaju podatke u JSON formatu.

Tako da nam ova linija baca grešku , a taman i da je nema, opet sa ovim ključevima ispod ne bi dobro izvukao podatke van.